change/@fluentui-react-nav-preview-b0930295-331e-4b55-96fc-c17b84ce854b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "[fix] - Speeding up animations in Nav.",
```

Comment:
most change files I see are in the format:
`fix: Speeding up animations in Nav.`

not a blocker but might be good for consistency